internal/app/init/main.go

```diff
@@ -165,6 +165,12 @@ func root() (err error) {
 		}
 	}
 
+	go func() {
```

Comment:
should this be here or in startSystemServices ( similar to dhclient )? overall indifferent, but probably worth defining a consistent location for these embedded services ( as in not containerized ). Or we could always toss this into an acpid container.

Reply:
We originally set out to make DHCP stuff a service, but currently its actually part of `init`. You bring up an interesting point though 🤔. dhclient and this are similar (not really a _service_ that a user or another node consumes) so for now it aligns with or current way of doing things. Maybe we implement a third way of start a service, perhaps in "in memory" one just to make it clear that services ran as containers are consumed by users/nodes? Just thinking out loud

Also I think this will have to stay as a go routine in `init` because eventually it will shutdown all containers and we would have to do some orchestration around not killing itself before getting to anything else.